### PR TITLE
Remove dead code [PD-1897]

### PR DESCRIPTION
**Pull request by tdphillips**
I'm currently going back through my modifications again to ensure I didn't break anything but it's looking like this may be it.

All tests pass.

### CODE REVIEW

File: seo/models.py [hunk 1]
```diff
@@ -1417,31 +1331,6 @@ class Meta:
         verbose_name_plural = 'Countries'
 
 
-class State(models.Model):
```
Comment (tdphillips) [inline seo/models.py]: State and City have no rows in them and aren't imported anywhere.

File: seo/models.py [hunk 2]
```diff
@@ -94,86 +93,7 @@ def clean(self, value, model_instance):
...
-    def prod_facets_for_current_site(self):
-        kwargs = {
-            'seositefacet__seosite__id': settings.SITE_ID,
-            'show_production': 1,
-        }
-        return self.filter(**kwargs)
-
-
 class CustomFacetManager(models.Manager):
-    def get_query_set(self):
```
Comment (tdphillips) [inline seo/models.py]: The custom queryset used here adds a new method that is never used so we can use the base queryset instead.

File: seo/cache.py [hunk 3]
```diff
@@ -24,14 +22,6 @@ def cache_page_prefix(request):
                             config.status, config.revision)
 
 
-def expire_site_view(host, path):
```
Comment (tdphillips) [inline seo/cache.py]: The commit that introduced this says it's fixing a regression but it's not used anywhere.

File: myjobs/decorators.py [hunk 4]
```diff
@@ -3,10 +3,10 @@
 from django.contrib.auth import logout
 from django.conf import settings
 from django.core.urlresolvers import reverse
-from django.http import Http404, HttpResponseRedirect, HttpResponseForbidden
+from django.http import HttpResponseRedirect, HttpResponseForbidden
 from django.shortcuts import get_object_or_404
 
-from myjobs.models import User, Activity, AppAccess
```
Comment (aspidites) [inline myjobs/decorators.py]: Ah. Cleaning up after my refactoring mess. Thanks/Sorry.

File: myjobs/helpers.py [hunk 5]
```diff
@@ -90,18 +89,6 @@ def log_to_jira(subject, body, issue_dict, from_email):
     return to_jira
 
 
-def make_fake_gravatar(name, size):
```
Comment (aspidites) [inline myjobs/helpers.py]: This is used in [./templates/mypartners/partner_reports.html](https://github.com/DirectEmployers/MyJobs/blob/quality-control/templates/mypartners/partner_reports.html#L51)

File: myjobs/templatetags/common_tags.py [hunk 6]
```diff
@@ -6,7 +6,7 @@
 
 from myjobs import version
 from myjobs.models import User
-from myjobs.helpers import get_completion, make_fake_gravatar
```
Comment (aspidites) [inline myjobs/templatetags/common_tags.py]: See comment above.

File: myjobs/templatetags/common_tags.py [hunk 7]
```diff
@@ -121,22 +121,6 @@ def get_gravatar(user, size=20):
         return ''
 
 
-@register.simple_tag
-def get_gravatar_by_id(user_id, size=20):
```
Comment (aspidites) [inline myjobs/templatetags/common_tags.py]: :+1:

File: myjobs/helpers.py (hunk 5 above)
Comment (tdphillips) [inline myjobs/helpers.py]: Re-added.

File: seo/forms/admin_forms.py [hunk 8]
```diff
@@ -280,19 +279,16 @@ def __init__(self, data=None, user=None, *args, **kwargs):
...
             form_field = self.fields[field]
 
+            inner['qs'] = form_field.queryset
             # If we're changing an existing SeoSite model, filter by its group.
             # Otherwise, filter as normal based on superuser status.
-            if field == 'special_commitments' or field == 'business_units' or\
-            field == 'featured_companies':
-                inner['qs'] = form_field.queryset
-            elif this:
+            if this and field not in ['special_commitments', 'business_units',
```
Comment (aspidites) [inline seo/forms/admin_forms.py]: Can we change "this" to "instance", since that seems to be the predominant convention elsewhere?

**Comment by aspidites** [comment]
:+1: Went through and found a couple of things, but things seem good now. Proof that large PRs *can* be gone through in a reasonable amount of time.

**Comment by koonsies** [comment]
:+1: hand tested a lot of modules. nothing jumped out of me from the code.

**Comment by tdphillips** [comment]
It's nice to know that my code is not a xenomorph.

**Pull request merged**